networkx/algorithms/approximation/tests/test_kcomponents.py

```diff
@@ -222,7 +222,7 @@ def setUp(self):
     def test_size(self):
         for G, A in self.GA:
             n = G.order()
-            s = len(G.edges())+len(A.edges())
+            s = len(list(G.edges())) + len(list(A.edges()))
```

Comment:
I think consistency is better in general.  Any of the `more_itertools` may be added to `itertlools` one day and then it will have been better to use the common names.